hack/local-up-cluster.sh

```diff
@@ -681,6 +716,9 @@ EOF
 fi
 }
 
+# let's check that etcd is in path before moving on
```

Comment:
alternately, factor out the test logic from `kube::etcd::start` and call it explicitly here?